wayland-client/src/event_queue.rs

```diff
@@ -30,14 +30,14 @@ use crate::{conn::SyncData, Connection, DispatchError, Proxy};
 ///
 /// In the rare case of an interface with *events* creating new objects (in the core protocol, the only
 /// instance of this is the `wl_data_device.data_offer` event), you'll need to implement the
-/// [`Dispatch::event_created_child()`] method. See the [`event_created_child!`](macro.event_created_child.html) macro
+/// [`Dispatch::event_created_child()`] method. See the [`crate::event_created_child!()`] macro
```

Comment:
Not sure if we want to get rid of `crate::` here and below?

Reply:
Yeah, displaying `crate::` in the documentation doesn't really make much sense. So using `[][]` to avoid display `crate::` seems like a good idea.